### ISSUE TITLE
java.lang.NullPointerException when using recursive decipher and same output directory as input

### ISSUE THREAD

**Issue by neothematrix** [Mikiya83/hbs_decipher#1]
It probably doesn't matter it's not a good practice to decipher the files in the same directory, but just for the sake of testing I did it :-)

The command I run was:
java -jar hybrid_backup_sync_decipher_1.3.jar -i Documents -o Documents -r -v

The "Documents" directory structure is:

```
Documents
- SubDirectory1
  --files--
- SubDirectory2
  -SubSubDirectory1
  -SubSubDirectory2
```

This is the error I got:

```
Enter in directory : SubDirectory2
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader.main(JarRsrcLoader.java:58)
Caused by: java.lang.NullPointerException
	at qnapdecrypt.QNAPFileDecrypter.decipherMultipleFiles(QNAPFileDecrypter.java:203)
	at qnapdecrypt.QNAPFileDecrypter.decipherMultipleFiles(QNAPFileDecrypter.java:223)
	at qnapdecrypt.QNAPFileDecrypter.main(QNAPFileDecrypter.java:193)
	... 5 more
```

**Comment by Mikiya83** [comment]
Indeed, i inverted two files name in this case, it should be fixed in eff00a6325569f76af45d3d748260573e8e1cf0f (and [ 892c192](https://github.com/Mikiya83/hbs_decipher_gui/commit/892c192c63372f2b7a516a924dfc3aa8bc759f9d) GUI version)
If you confirm it's fixed for you too in 1.3.1, i can close this issue.

**Comment by neothematrix** [comment]
Sorry for the late reply, for some reason I don't receive github notification email...
I just tested it, it works wonderfully now!
I noticed a small typo in the GUI panel, the title description (not the window title) says "Hybric" instead of "Hybrid", I know I'm picky :-D

**Comment by Mikiya83** [comment]
Fix done in 1.3.2 (with small minor fixes)
Thanks for the report ! ;)
